users/models.py

```diff
@@ -206,3 +206,11 @@ def update_remove_inactive_user_from_projects(sender: "CustomUser", **kwargs: An
     if not user.is_active:
         user.projects.clear()
         logger.debug(f"User: {user.pk} has been removed from all projects")
+
+
+@receiver(post_save, sender=CustomUser)
+def create_custom_user_preferences(sender: "CustomUser", created: bool, **kwargs: Any) -> None:
```

Comment:
If you're using `CustomUser` class above you don't need to add typing in quotes here

Reply:
Done